t/proxyunits.t

```diff
@@ -49,6 +52,7 @@ sub check_version {
     like(<$ps>, qr/VERSION /, "version received");
 }
 
+announce("Initialization", __LINE__);
```

Comment:
please remove `announce` and use the builtin `note()` if you feel like that information is useful. print outputs are selectively eaten while note is explicit.